src/main/kotlin/lib/graphics/turtle/awt/GraphicsTurtleCore.kt

```diff
@@ -15,14 +16,17 @@ import kotlin.math.sin
  * [TurtleCore] interface over AWT [Graphics2D]
  */
 class GraphicsTurtleCore(
-    graphics: Graphics2D,
+    graphics: Graphics,
     private val size: Dimension,
     private val stateView: TurtleStateView
 ) : TurtleCore {
     private val graphics = object {
+        // Every Graphics in runtime is actually Graphics2D
+        private val g2d = graphics as Graphics2D
+
         // Force all interactions with Graphics2D to happen in  UI thread
         operator fun invoke(block: Graphics2D.() -> Unit): Unit = invokeAndWait {
-            graphics.block()
+            g2d.block()
         }
     }
```

Comment:
Предлагаю весь этот `object` со всеми потрохами поменять на:
```suggestion
    // Every Graphics in runtime is actually Graphics2D
    private val graphics = graphics as Graphics2D
    // Force all interactions with Graphics2D to happen in  UI thread
    private operator fun Graphics2D.invoke(block: Graphics2D.() -> Unit) = invokeAndWait { block() }
```

Reply:
Поправил. @gumanoid глянешь?